examples/angular/src/pages/ui/components/authenticator/auth-with-translations/auth-with-translations.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component, OnInit } from '@angular/core';
+import Amplify, { I18n } from 'aws-amplify';
+import { translations, DefaultTexts } from '@aws-amplify/ui';
+import awsExports from '@environments/auth-with-email/src/aws-exports';
+@Component({
+  selector: 'auth-with-translations',
+  templateUrl: 'auth-with-translations.component.html',
+})
+export class AuthWithTranslationsComponent implements OnInit {
+  constructor() {
+    Amplify.configure({
+      ...awsExports,
+      auth: {
+        login_mechanisms: ['email'],
+      },
+    });
+  }
+
+  ngOnInit() {
+    I18n.putVocabularies(translations); // TODO: this should be done by amplify
+    I18n.setLanguage('ja');
+
+    // Provide missing translations
+    I18n.putVocabulariesForLanguage('ja', {
+      [DefaultTexts.CONFIRM_PASSWORD_LABEL]: 'パスワードの確認',
+      [DefaultTexts.CONFIRM_SMS_LABEL]: 'SMSコードを確認する',
+      [DefaultTexts.CONFIRM_TOTP_LABEL]: 'TOTPコードを確認する',
+    });
```

Comment:
This is how customer can put custom translations using `DefaultTexts` constant. This helps them be consistent with what we are using.